### ISSUE TITLE
[feat]: Use unique, non-.local Alternative Name to Embassy cert for downstream access

### ISSUE THREAD

**Issue by cgalladora** [Start9Labs/start-os#1313]
### Prerequisites

- [X] I have searched for [existing issues](https://github.com/start9labs/embassy-os/issues) that already suggest this feature, without success.

### Describe the Feature Request

Statement of facts and problem:
Over LAN, users will typically access their Embassy using some "common name" .local, e.g. embassy-########.local, which uses mDNS to become available on its' network
When the Embassy is sitting in a DMZ or upstream network from the endpoints, mDNS seems to not work. The ostensible solution is to add embassy-########.local to the DNS of the downstream network, but many DNS servers don't play nicely with .local
Currently you can assign embassy.embassy to the device IP on your DNS and it works exactly as expected (yay), since embassy.embassy is an Alternate Name on the cert.

In the future, however, if multiple embassies are on the subnet/DMZ, this won't work since embassy.embassy will not be a unique name. 

### Describe the Use Case

I have two Embassy devices on my DMZ. I want to access them from my main network over LAN. Since .local won't work (unless we get into wide mDNS- not recommended), if I want to access them using a name on the cert, I have to use embassy.embassy, but which device responds to embassy.embassy, and how do I access the other one?

### Describe Preferred Solution

Replace the embassy.embassy Alternative name with something like "embassy-########.embassy", where ####### represents the unique id typically seen on the .local address. 

### Anything else?

In my opinion this is a fairly unique circumstance. Thanks as always to the incredible Start9 team.

**Comment by ProofOfKeags** [comment]
TL;DR: Allow users to add additional SANs to the certificate that is used on the main Embassy dashboard.

[Relevant Code](https://github.com/Start9Labs/embassy-os/blob/master/backend/src/net/ssl.rs) for anyone who wants to tackle this.
